src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -0,0 +1,483 @@
+
+namespace PowerAccent.Core
+{
+    using System;
+    using PowerToys.PowerAccentKeyboardService;
+
+    public enum Language
+    {
+        ALL,
+        CUR,
+        CZ,
+        DE,
+        FR,
+        GR,
+        IS,
+        MI,
+        PI,
+        PL,
+        SK,
+        SP,
+        TK,
+    }
+
+    internal class Languages
+    {
+        public static char[] GetDefaultLetterKey(LetterKey letter, Language lang)
+        {
+            switch (lang)
+            {
+                case Language.ALL: return GetDefaultLetterKeyALL(letter); // ALL
+                case Language.CUR: return GetDefaultLetterKeyCUR(letter); // Currency
+                case Language.CZ: return GetDefaultLetterKeyCZ(letter); // Czech
+                case Language.DE: return GetDefaultLetterKeyDE(letter); // German
+                case Language.FR: return GetDefaultLetterKeyFR(letter); // French
+                case Language.GR: return GetDefaultLetterKeyGR(letter); // Greek
+                case Language.IS: return GetDefaultLetterKeyIS(letter); // Iceland
+                case Language.MI: return GetDefaultLetterKeyMI(letter); // Maori
+                case Language.PI: return GetDefaultLetterKeyPI(letter); // Pinyin
+                case Language.PL: return GetDefaultLetterKeyPL(letter); // Polish
+                case Language.SK: return GetDefaultLetterKeySK(letter); // Slovak
+                case Language.SP: return GetDefaultLetterKeySP(letter); // Spain
+                case Language.TK: return GetDefaultLetterKeyTK(letter); // Turkish
+            }
+
+            throw new ArgumentException("The language {0} is not know in this context", lang.ToString());
+        }
+
+        // All
+        private static char[] GetDefaultLetterKeyALL(LetterKey letter)
+        {
+            switch (letter)
+            {
+                case LetterKey.VK_0:
+                    return new char[] { '₀', '⁰' };
+                case LetterKey.VK_1:
+                    return new char[] { '₁', '¹' };
+                case LetterKey.VK_2:
+                    return new char[] { '₂', '²' };
+                case LetterKey.VK_3:
+                    return new char[] { '₃', '³' };
+                case LetterKey.VK_4:
+                    return new char[] { '₄', '⁴' };
+                case LetterKey.VK_5:
+                    return new char[] { '₅', '⁵' };
+                case LetterKey.VK_6:
+                    return new char[] { '₆', '⁶' };
+                case LetterKey.VK_7:
+                    return new char[] { '₇', '⁷' };
+                case LetterKey.VK_8:
+                    return new char[] { '₈', '⁸' };
+                case LetterKey.VK_9:
+                    return new char[] { '₉', '⁹' };
+                case LetterKey.VK_A:
+                    return new char[] { 'á', 'à', 'ä', 'â', 'ă', 'å', 'α', 'ā', 'ą', 'ȧ', 'ã', 'æ' };
+                case LetterKey.VK_B:
+                    return new char[] { 'ḃ', 'ß', 'β' };
```

Comment:
isn't the second one the `ß` from Germany? Then it has to be moved to the letter `s`.

Reply:
He knows :) https://github.com/damienleroy/PowerAccent/issues/42